services/dask-sidecar/docker/boot.sh

```diff
@@ -34,19 +34,19 @@ fi
 #
 
 if [ ${DASK_START_AS_SCHEDULER+x} ]; then
-  SCHEDULER_VERSION=$(dask-scheduler --version)
-
-  echo "$INFO" "Starting as ${SCHEDULER_VERSION}..."
+  scheduler_version=$(dask-scheduler --version)
```

Comment:
Just to clarify, do you suggest [the following convention](https://unix.stackexchange.com/questions/42847/are-there-naming-conventions-for-variables-in-shell-scripts)?

>>Environment variables or shell variables that are introduced by the operating system, shell startup scripts, or by the shell itself etc. are usually all in CAPITALS.
>>
>>To prevent your own variables from conflicting with these variables, it is a good practice to use lower_case variable names.